edd-activecampaign.php

```diff
@@ -588,26 +593,32 @@ public function add_metabox() {
 		}
 	}
 
-	/**
-	 * Render the metabox displayed on the Download edit screen.
-	 *
-	 * @since  1.1
-	 * @access public
-	 */
-	public function render_metabox() {
-		global $post;
-
-		echo '<p>' . __( 'Select the lists you wish buyers to be subscribed to when purchasing.', 'edd-activecampaign' ) . '</p>';
-
-		$checked = (array) get_post_meta( $post->ID, '_edd_activecampaign', true );
-
-		foreach ( $this->get_lists() as $list_id => $list_name ) {
-			echo '<label>';
-				echo '<input type="checkbox" name="_edd_activecampaign[]" value="' . esc_attr( $list_id ) . '"' . checked( true, in_array( $list_id, $checked ), false ) . '>';
-				echo '&nbsp;' . $list_name;
-			echo '</label><br/>';
+		/**
+		 * Render the metabox displayed on the Download edit screen.
+		 *
+		 * @since  1.1
+		 * @param object $post The Post object.
+		 * @access public
+		 */
+		public function render_metabox( $post ) {
+			echo '<fieldset class="edd_activecampaign_lists edd-form-group">';
+			echo '<legend><p>' . esc_html__( 'Select the lists you wish buyers to be subscribed to when purchasing.', 'edd-activecampaign' ) . '</p></legend><div id="edd_activecampaign_checkbox_lists">';
+
+			$checked      = (array) get_post_meta( $post->ID, '_edd_activecampaign', true );
+			$checked_list = implode( ',', $checked );
+			foreach ( $this->get_lists() as $list_id => $list_name ) {
+				echo '<div class="edd-form-group__control">';
+					echo '<input type="checkbox" name="_edd_activecampaign[]" class="edd-form-group__input" id="' . esc_attr( $list_id ) . '"value="' . esc_attr( $list_id ) . '"' . checked( true, in_array( $list_id, $checked ), false ) . '>';
```

Comment:
```suggestion
					echo '<input type="checkbox" name="_edd_activecampaign[]" class="edd-form-group__input" id="' . esc_attr( $list_id ) . '" value="' . esc_attr( $list_id ) . '"' . checked( true, in_array( $list_id, $checked ), false ) . '>';
```
I think this is the last thing--and it's working without it but I feel like the space should really be there before the value attribute.